seismic-app/src/styles/_blog.scss

```diff
@@ -0,0 +1,107 @@
+h1 {
+  font-family: 'Titillium Web', sans-serif;
```

Comment:
While the previous front-end was using Titillium, I think we should use Open Sans, as it is mentioned in the [specs](https://www.figma.com/file/qOyFrU8T6LXnJw7YcoVW85/Seismic-Risk?node-id=0%3A1)
The main reason to do this is because it seems it doesn't work very well with some romanian characters (e.g. 'ș' from [Bucureștiul Vulnerabil](https://github.com/code4romania/seismic-risc-client/pull/20/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.scss#diff-cff496b8c25a449e7af680389bc111bdR17))

Reply:
Seems I should have checked the specs a little better since the font is the proper one
Will be investigating the `ș` problem some other time if it's not a local one